apis/kueue/v1beta1/localqueue_types.go

```diff
@@ -51,13 +56,21 @@ type LocalQueueStatus struct {
 	// +patchMergeKey=type
 	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
 
-	// flavorUsage are the used quotas, by flavor currently in use by the
+	// flavorsReservation are the reserved quotas, by flavor currently in use by the
+	// workloads assigned to this LocalQueue.
+	// +listType=map
+	// +listMapKey=name
+	// +kubebuilder:validation:MaxItems=16
+	// +optional
+	FlavorsReservation []LocalQueueFlavorUsage `json:"flavorsReservation"`
+
+	// flavorsUsage are the used quotas, by flavor currently in use by the
 	// workloads assigned to this LocalQueue.
 	// +listType=map
 	// +listMapKey=name
 	// +kubebuilder:validation:MaxItems=16
 	// +optional
-	FlavorUsage []LocalQueueFlavorUsage `json:"flavorUsage"`
+	FlavorsUsage []LocalQueueFlavorUsage `json:"flavorsUsage"`
```

Comment:
```suggestion
	FlavorUsage []LocalQueueFlavorUsage `json:"flavorUsage"`
```

Oh, we should use the `flavorsUsage`. However, we can not fix this name along with a breaking change since our API version is beta.

Reply:
True :(
I added it to the wishlist #768